### DISCUSSION TITLE
Remove thermostat schedule hold

### DISCUSSION THREAD

**Discussion by folotp** [d0n4v4nb3ck3r/homebridge-honeywell-home#782] [Ideas]
Is there a way to return the thermostat to running the current program? That is, after having made changes to the setpoint or fan mode, tell the thermostat to cancel all that and return to the current program according to schedule (day, nigh, away...)

**Comment by folotp** [comment]
Thermostats that are better integrated with Homekit, such as the Ecobees, have a switch to remove the hold. You can thus create a scene to remove the hold. Would it be possible to add such a switch that would set the API command "thermostatSetpointStatus" : "NoHold"?

I also think that maybe you can get the hold status using the Get Thermostat command and data points changeableValues.thermostatSetpointStatus and changeableValues.holdUntil.

**Reply by donavanbecker** [reply]
Please open Feature Request issue for better tracking and I can look into it.
